Makefile

```diff
@@ -95,3 +99,15 @@ undeploy-spire-server: check-cluster
 .PHONY: check-spire-server
 check-spire-server: check-cluster
 	@$(CHECK_SPIRE_SCRIPT)
+
+.PHONY: deploy-spire-agent
+deploy-spire-agent: certs
+	@$(DEPLOY_SPIRE_AGENT_SCRIPT)
+
+.PHONY: undeploy-spire-agent
+undeploy-spire-agent:
+	@$(UNDEPLOY_SPIRE_AGENT_SCRIPT)
+
+.PHONY: env-down
+env-down: undeploy-spire-agent undeploy-spire-server cluster-down
```

Comment:
@cursoragent cluster-down target should just automatically clean up the server and agents, right?